src/python/WMCore/MicroService/MSPileup/MSPileupReport.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+"""
+File       : MSPileupReport.py
+Author     : Valentin Kuznetsov <vkuznet AT gmail dot com>
+Description: MSPileup report module
+"""
+
+# WMCore modules
+from Utils.Timers import gmtimeSeconds, encodeTimestamp
+
+
+class MSPileupReport():
```

Comment:
This class looks more like a data structure implementation, so we might want to move it under the MSPileup/DataStrucs package.

Reply:
done